Sources/TuistGraph/Models/Dependencies/CarthageDependencies.swift

```diff
@@ -74,3 +80,21 @@ extension CarthageDependencies {
         }
     }
 }
+
+// MARK: - CarthageDependencies.Options
+
+extension CarthageDependencies {
+    public enum Options: String {
```

Comment:
2 things:
- you are missing the corresponding `Options` in the `ProjectDescription` target
- you should use an enum, similar to what we are doing for `Project.Options`

Reply:
Instead of an enum with cases and then take a set of it, it should be a struct, and the cases would become boolean property of the struct